src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -977,6 +962,7 @@ export class HaConfigDevicePage extends LitElement {
   }
 
   private _getDeleteActions() {
+    this._deleteButtons = [];
```

Comment:
Should we only reset if the device is undefined?

Reply:
`_getDeleteActions` is only called on device changes but for consistency and readability I'll just reset everything on device change instead of in the individual methods